deploy/kubernetes/helm/mcp-apache-spark-history-server/templates/externalsecret.yaml

```diff
@@ -0,0 +1,30 @@
+{{- if and .Values.auth.enabled .Values.auth.externalsecret.create }}
```

Comment:
can you add basic tests for this file? 

Example: https://github.com/DeepDiagnostix-AI/mcp-apache-spark-history-server/blob/main/deploy/kubernetes/helm/mcp-apache-spark-history-server/tests/service_test.yaml